models/public/yolo-v4-tf/accuracy-check.yml

```diff
@@ -1,64 +1,4 @@
 models:
-  - name: yolo-v4-tf
```

Comment:
Is there no way to validate the SavedModel version through AC?

Reply:
validation in source framework returned